apis/Google.Cloud.EntityFrameworkCore.Spanner/Google.Cloud.EntityFrameworkCore.Spanner/Diagnostics/SpannerLogBridge.cs

```diff
@@ -20,7 +20,7 @@ namespace Microsoft.EntityFrameworkCore.Diagnostics
     /// <summary>
     /// This is internal functionality and not intended for public use.
     /// </summary>
-    public class SpannerLogBridge<TLoggerCategory> : DefaultLogger
+    public class SpannerLogBridge<TLoggerCategory> : Logger
```

Comment:
I think this one can be `internal` as well, it's only being used on the EF provider project.

Reply:
Possibly, but it's harder to tell - it may be that users are expected to instantiate it themselves. (It's not in an "Internal" folder, for example.)

I'm not overly worried about the EF provider at the moment - it's very alpha, and may want a larger overhaul before going any further anyway :)